src/components/dashboard/dashboardCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { SubCard, Content, Card, Image, Information } from "./dasboardStyle.jsx" 
+import logo from './waterloo-logo.jpg'
+
+export default function DashboardCard({ value, ...props }) {
```

Comment:
do you think we can reduce the card height? It feels like a lot of scrolling for the 4 universities on there so far